clientapp/EntryPoint.cpp

```diff
@@ -379,7 +372,8 @@ int main(int argc, char * argv[])
 	}
 	else
 	{
-		auto mmenu = CMainMenu::create();
+		bool playIntroVideo = !settings["session"]["headless"].Bool() && !vm.count("battle") && !vm.count("nointro") && settings["video"]["showIntro"].Bool();
+		auto mmenu = CMainMenu::create(playIntroVideo);
```

Comment:
Perhaps call intro playback separately and remove that bool parameter? Should make code more clear:
```suggestion
		bool playIntroVideo = !settings["session"]["headless"].Bool() && !vm.count("battle") && !vm.count("nointro") && settings["video"]["showIntro"].Bool();
		auto mmenu = CMainMenu::create();
		if (playIntroVideo)
			mmenu->playIntro();
```